src/main/java/org/broadinstitute/hellbender/tools/exome/ExonCollection.java

```diff
@@ -42,6 +43,24 @@
     E exon(final int index);
 
     /**
+     * Returns the name of an exon.
+     *
+     * <p>
+     *     This method is guaranteed to return the name of the exon in this collection as long as it is indeed
+     *     included in it.
+     * </p>
```

Comment:
On further reflection.  Leave the javadoc however you like it.  We should let @vdauwera decide what the right formatting of this sort of thing is.